### ISSUE TITLE
Removal of death benefits makes capturing too powerful

### ISSUE THREAD

**Issue by Bladewood** [endless-sky/endless-sky#2283]
Now that death benefits are gone, there is nothing to deter players from capturing ships as much as they want. And while depreciation may nerf the money-generating aspect, it does nothing to prevent players from capturing ships for actual combat purposes.

**Comment by Lineth** [comment]
the most reliable source of capturable ships is t1 pirate ships which are cannon fodder at t1.5+. Unfettered tear them apart.

IMO its fine, as it was always easy to amass a large fleet. old ways you just stole triple plasmas and bought them in bulk.

**Comment by KiLEdEnNis** [comment]
not that I'm picking a side, but a fact is that amassing a T1 fleet early is way easier that TPC farming

**Comment by MessyMix** [comment]
Bladewood, I completely agree and it's something I have strongly believed.

**Comment by KiLEdEnNis** [comment]
death benefits cost ya stuff already when capping, depreciation only when selling
what if death benefits come back in a nerfed state & we nerf depreciatiion as well? so ya lose a bit of money when capping & make a bit less profit

**Comment by Lineth** [comment]
The problem with death benefits was that it incentivized going after small ships instead of the large ones, which wasn't what it was supposed to do. Personally I hated it as a mechanic and do not mourn its loss.

**Comment by Tadrix** [comment]
The problem with death benefits was, as TVtropes put it, that
> The amount scales in proportion to the captured ship's value, so you'll never have to pay more than the ship is worth, but it only takes a few casualties before your slice of the pie becomes razor-thin.

Because small ships can be boarded with no casualties, this incentivized going after small ships. Personally, I have opinion that depreciation to 25% is too harsh, 50% over two years would suffice (so I don't like depreciation as it is now). As for amount of death benefits, I proposed a different formula in #2154 .

None of these mechanics are something to be liked and they don't really help to balance a merchant fleet of 500 Bactrians (which in this game is practical, but boring), but majority here seems to support the opinion that a single flagship capping cheap ships/outfits is still imbalanced.

**Comment by wambat1** [comment]
In previous iterations I could amass enough wealth to buy my Bactrian x4 and SBs x12 within a couple hours playtime, and a couple months game-time. In the current version it has take me over 10 days playtime to get to that point, and I have been forced to capture most of my SB fleet.

One of the issues with death bennefits is that you didn't actually have to pay them, so you could have access to extreme amounts of capital and loans early on which could put you in a Bactrian in a few weeks game-time, and a lot less than that if you ignored the starting missions. From there your income could advance exponentially.

**Comment by LocalGod79** [comment]
as it stands, even using a Bactrian to capture ships en-masse, when you take casualties, you reduce the number of additional potential ships that you can capture - and the curve is exponential, because you're more likely to take casualties with the reduced crew.

in-game, this means that with a boarding-outfitted Clipper the daily profit margins are not really any different than they were before depreciation.

all of that said, the bank represents some code that should be taken advantage of, and perhaps a deductable should be paid for crew member's life insurance claims...

**Comment by MessyMix** [comment]
Fair point, although keep in mind that there is no longer a penalty for
keeping captured ships now; previously death benefits punished the player
for doing so, meaning that I would not capture ships unless I wanted to
keep them.

**Comment by KiLEdEnNis** [comment]
again; bring back death benefits but tune down both :3
(it would also encourage choosing between keeping or selling, I prefer selling & do so)

**Comment by DingusShingleton** [comment]
Before deprecation came along I suggested profit sharing in an issue that received a pretty good response. I later wrote up a much more concrete proposal, but by this time deprecation had come up and I figured eh what the heck. But maybe given this discussion there's still room to work it in? I'll post it here for your consideration (please ignore the parts of "Existing Mechanics" that are no longer accurate):
___
*in which dingus blathers on at length about plunder/capture balance and yet again can offer no help with the actual coding*

*to skip the explanation go straight to the bullet points*

### Preface
There has been a recurring topic of discussion on the various forums of this game that revolves around the balance of capturing and plundering (for the purposes of this post, “plundering” refers to boarding a ship and stealing its outfits, and “capturing” to “attempt[ing] to capture” it until all its crew members are dead and this ship changes owners).

To be sure, both of these have great potential to be tremendous money makers; plundering is basically a no-risk proposition, making it all upside, while capturing, though it can have great costs, can also be the most lucrative profession in the game (short of the exponential trader, but that’s balanced in other ways).

Previously, I posted an issue in which I discussed these ideas, but they were newly formed and rather meandering. In the intervening time I have put great thought into this, and I think I now have a much more coherent and purposeful proposal in mind.

### Existing Mechanics
First, the game as it stands. As I said, plundering is more or less no-risk, but capturing is balanced using two mechanics: the death benefit and (to a lesser extent) crew salary. I believe these to be inherently flawed on a conceptual level. Capturing a vessel, with a little bit of know-how, quickly becomes almost as no-risk as plundering; at that point paying benefits is irrelevant, because it is based on something that may or may not occur: crew death (and even if it does, it usually just lowers your profit from outrageous to reasonable). Furthermore, a moment of reflection reveals that, as a pilot, I am liable to pay benefits for crew members who die in boarding actions… but not for those who die in ship to ship combat! Huh?

Crew salary, on the other hand, is a modest and static fee. It doesn’t scale at nearly the same rate that profit from capturing does: taking a sparrow requires maybe 4 or 5 crew with lasers to be a sure-fire thing, netting the pilot a few hundred thousand credits. This one tiny ship pays for those 4 extra crew for at least 500 days. A pretty good deal I’d reckon, especially considering that, crucially, *the pilot risks nothing in a boarding action*. The crew get 100 credits a day, live or die. That’s not how markets work.

### ~~Death Benefit~~ Profit Sharing
So. My first modification would be to outright eliminate the death benefit. It doesn’t make sense internally and is ultimately quite ineffective at its goal. In its place I would introduce profit sharing. 

- Replaces death benefit
- Whenever the player steals outfits or captures a ship, the total current market value of the loot is split between the remaining crew members and the pilot, who of course gets the lion’s share. I think something like a 50/50 split would work nicely: 50% to the pilot and 50% between everyone else. 
- Each instance of profit sharing would then be added to a 0% (or higher?) loan in the bank tab, which could be payed off at the player’s leisure. The crewmembers understand that you have to sell the newly acquired goods before they can get their payment

 This way, the balancing factor is predicated on something which is *guaranteed* to take place, every time, and which scales perfectly no matter how big or small the take is. Clean, simple, effective.

Now, you might be thinking “but those poor dead boarding crew, what about their families?! You’re a heartless monster!”. Tell that to the dozens of crew that die every time a capital ship is destroyed. Their families get nothing. I think that in the universe of ES, it is perfectly reasonable to assume that if you are joining a pilot’s fleet, there is the real risk of death. It’s part of the job, regardless of how it happens, but the rewards for taking that risk can be huge. This brings be nicely to my second point:

### Salary
As I mentioned before, I believe that crew salary should scale as the player loses crew. 

- Every time an escort ship is destroyed or a crew member dies while boarding, it is tallied up, much in the way combat rating is tallied. 
- This applies a flat modifier to the daily upkeep of paying your crew, perhaps on an exponential scale. 

Have a penchant for sending fresh faced crew members to their deaths? That’s swell. But good luck hiring fresh meat at 100/day. The actual numbers are more or less irrelevant to the player, but it presents another opportunity to earn titles: get enough crew members killed, and you become known as... _The Butcher_.

Of course, I understand that fun is paramount, and for this reason I think that:

1. The crew mortality rating should, like bank loans, take only a trailing total for a certain time period (thanks @ldsgsm for that idea). If you straighten out and start caring about your crew’s lives, the costs will come back down. And... 
2. The increase, exponential or otherwise, should be fairly forgiving. I don’t want to bring the hammer down on players who like to play fast and loose with their crews’ lives, but I think that that playstyle should be acknowledged in some meaningful way. Even with these changes, plundering and looting will still be profitable, and this just serves as a nice additional smoothening factor.

### Conclusion
In conclusion, I believe that these (relatively) simple changes would basically solve those pesky balance issues lickety-split, while also adding a modest dose of realism and immersion to the game. I must also of course disclaim that I am unable to code this myself, but I feel very strongly that if someone were to give it a chance, it would be very well received.

**Comment by Tadrix** [comment]
The problem with depreciation is this:
> (it would also encourage choosing between keeping or selling, I prefer selling & do so)

Yeah, well, I also like to sell. And then buy some brand new ship (which smells like it just came out of factory and has floors which hadn't had to be wiped clean from all that blood. Previous system allowed it; now it's getting harder (I'd need to sell 4 times as many ships to buy one). I'm ok with used ship vs. new ship price difference, but this difference is very large — if used equipment costed 50% of original price, it would be much better.

> Now, you might be thinking “but those poor dead boarding crew, what about their families?! You’re a heartless monster!”. Tell that to the dozens of crew that die every time a capital ship is destroyed. Their families get nothing. I think that in the universe of ES, it is perfectly reasonable to assume that if you are joining a pilot’s fleet, there is the real risk of death.

Agreed. There's only one reason to hire extra crew: to board other ships. If they take the job knowing that piloting or gunnery won't be their everyday activity, but that rifle and grenade pack is their best and only friend, then they signed up for this. And they don't need much higher salary then other crew, seeing as mostly they are just sitting there and become useful only during/after major battle.

**Comment by DingusShingleton** [comment]
> There's only one reason to hire extra crew: to board other ships.

This is true and it's a reasonable line of thought.... but remember, extra crew are not the only crew you hire, nor are they the only crew that take part in boarding actions. Every single crew member, extra or not, is potentially liable to die while on the job, whether while boarding or simply flying in dangerous waters.

**Comment by VonZorch** [comment]
Add the ability to select the size of an offensive boarding party, this would use extra crew first.
Extra crew who are purely boarders should be paid less say 50 Cr/day as they just sit around eating your groceries, telling war stories and playing video games (they call it training), but when you dock on a new ship and attempt capture every member of the boarding party gets a combat bonus, with depreciation 100Cr sounds right.
Defensive boarding actions will of course use the full crew and maybe a random percentage of the passengers.

**Comment by gunqqer** [comment]
Passengers will have zero training and be on the back lines, they would not be doing anything unless *all* other crew is ded

**Comment by KiLEdEnNis** [comment]
only attack with a fraction of your crew? the other ship uses full crew so if you use less crew you get a disadvantage: you kill less enemies and thus lose more crew

**Comment by wambat1** [comment]
Reality check:
a) I don't think there is any reason to believe specially trained combat troops would be less expensive to retain than regular crew. So far as I know, in the historical high-seas naval warfare most space games like this one take their inspiration from, marines were more expensive, and it was the regular crew that was more expendable; and yes, everybody fought in a boarding action.
b) If we take our inspiration more from the Star Trek example, which I think is more helpful, we see that generally all crew is well cross trained for most roles. The reality is that you have just tons of time in space travel, and not much else to do with it. So, again, better represented by averaged salaries and all crew taking part in boarding actions.
c) If we differentiate crew, such that all excess crew are soldiers not capable of operating a ship, then every ship captured will necessarily rob the flagship of essential crew and cause unresponsiveness after the first capture.  

    On Tuesday, April 4, 2017 12:04 AM, KiLEdEnNis <notifications@github.com> wrote:
 

 only attack with a fraction of your crew? the other ship uses full crew so if you use less crew you get a disadvantage: you kill less enemies and thus lose more crew—

**Comment by DingusShingleton** [comment]
For the record, I completely agree that differentiating crew is a very difficult road to travel. Without a fully fledged crew system, a half baked implementation of multiple crew types would be a disaster. My proposal called for the increase of crew salaries across the board as the death count rises, as getting into the nitty gritty about who's doing what just isn't worth it imo.

**Comment by ldsgsm** [comment]
I agree, with the caveat that a two-tier system of "marines are the same as
other crew except they cost more and have somewhat better stats" would also
be okay.

**Comment by KiLEdEnNis** [comment]
I kinda like that the game has a lot of simple mechanics like crew, if we complicate everything it could make the game more "grindy"

**Comment by SirStigII** [comment]
I like the idea of a profit share, although I think you could slim down the concept to just being a % taken out of your profits when sold.

**Comment by DingusShingleton** [comment]
Then it only affects you if you sell the ships. If you keep them it's a complete free roll

**Comment by SirStigII** [comment]
You could still have to pay the percentage whenever you install them to your ship. Basically as soon as they leave cargo space you would owe them their cut, and/or the first planet with a shipyard that you land at for captured ships.

**Comment by DingusShingleton** [comment]
That honestly seems more complicated, and could potentially punish players for keeping ships with a massive one time payment. Death benefits was payed out as a loan, I don't see why this couldn't also be. It just makes sense from an in-game perspective, and game mechanically, to not demand a huge payment up front. In my opinion, anyway

**Comment by DingusShingleton** [comment]
The thing is, the second that property becomes yours, your net worth shoots up. You are worth that much more in total, but that doesn't mean you have that value in a liquid format. Sharing your non liquid assets with crew members isn't feasible, so the only option is to compensate them with cash. In times when large sums of cash might not be readily available, loans are preferable. This would all (presumably) be laid out in the terms of the crews' contracts, and the profit share payments would simply be bonus salary that they would collect each day in addition to their normal paycheck.

**Comment by VonZorch** [comment]
Is this profit sharing or prize money?
If it's profit sharing the crew would get a cut of your entire net worth change, if positive, including pay for missions and net profit after buy selling then buying cargo.  In that case base pay needs to drop or disappear altogether as the crew works for shares .
Prize money would be the value of acquisitions from salvage and bounty payments, with no change in base pay.
One historic model is owner gets 50% the rest gets split among the rest of the crew.  That assumes the Captain is the owner as is the case in game.
